javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+let count = 4;
+
+function updateCountdown() {
+  if (count === 0) {
+    $countdownDisplay.textContent = `~Earth Beeeelooowww~`;
+    clearInterval();
```

Comment:
`clearInterval` should be getting called with 1 argument

Reply:
![Screenshot 2023-06-05 at 2 12 24 PM](https://github.com/j-alejandro-araujo/c0523-code-solutions/assets/91649194/a5ec0759-80c0-4e45-acec-dfff2be1eda4)

Adjusted!